Packs/Infoblox/Integrations/Infoblox/Infoblox.yml

```diff
@@ -791,6 +803,231 @@ script:
+      required: false
+    outputs:
+    - contextPath: Infoblox.Host.ReferenceID
+      description: The host record reference ID.
+      type: String
+    - contextPath: Infoblox.Host.IPV4Addresses
+      description: The host IPv4 addresses.
+      type: String
+    - contextPath: Infoblox.Host.IPV6Addresses
+      description: The host IPv6 addresses.
+      type: String
+    - contextPath: Infoblox.Host.ConfigureForDNS
+      description: Whether the host is configured for DNS.
+      type: Boolean
+    - contextPath: Infoblox.Host.Name
+      description: The host record name.
+      type: String
+    - contextPath: Infoblox.Host.ExtendedAttributes
+      description: The network extended attributes.
+      type: Unknown
+    - contextPath: Infoblox.Host.AdditionalFields
+      description: The additional fields for network.
+      type: Unknown
+    - contextPath: Infoblox.Host.Aliases
+      description: The aliases for this host record.
+      type: String
+    - contextPath: Infoblox.Host.AllowTelnet
+      description: Whether the host is allowed to use telnet.
+      type: Boolean
+    - contextPath: Infoblox.Host.NetworkView
+      description: The network view of the host record.
+      type: String
+    - contextPath: Infoblox.Host.View
+      description: The view of the host record.
+      type: String
+    - contextPath: Infoblox.Host.Zone
+      description: The zone of the host record.
+      type: String
+    - contextPath: Infoblox.Host.Disable
+      description: Whether the host is disabled.
+      type: Boolean
+    - contextPath: Infoblox.Host.DNSName
+      description: The DNS name of the host record.
+      type: String
+  - name: infoblox-dhcp-lease-lookup
+    description: Look up a DHCP lease.
+    arguments:
+    - name: ip_address
+      description: Lease IP address (IPv4 or IPv6).
+      required: false
+    - name: hardware
+      description: MAC address for IPv4 leases. Regex or exact search supported.
+      required: false
+    - name: hostname
+      description: Hostname sent via DHCP option 12. Regex/exact search.
+      required: false
+    - name: ipv6_duid
+      description: IPv6 DUID identifier for IPv6 leases. Regex/exact search.
+      required: false
+    - name: protocol
+      auto: PREDEFINED
+      predefined:
+      - IPV4
+      - IPV6
+      - BOTH
+      defaultValue: BOTH
+      description: The protocol of the DHCP lease.
+      required: false
+    - name: fingerprint
+      description: DHCP client fingerprint; case-insensitive or regex search.
+      required: false
+    - name: username
+      description: Specify the user associated with the lease request.
+      required: false
+    - name: limit
+      defaultValue: 100
+      description: Specify the maximum number of leases to return.
+      required: false
+    outputs:
+    - contextPath: Infoblox.DHCPLease.ReferenceID
+      description: The DHCP lease reference ID.
+      type: String
```

Comment:
In the outputs for the `infoblox-dhcp-lease-lookup` command, several `type` values are capitalized (e.g., `String`, `Boolean`). Per XSOAR standards, these should be in lowercase (`string`, `boolean`). Please correct the types for all outputs in this command from line 985 to 1030.